src/rprblender/engine/context.py

```diff
@@ -220,6 +224,28 @@ def resize(self, width, height):
         if composite:
             self._enable_catchers()
 
+    def sync_cryptomatte_hash(self):
+        view_layer = self.blender_data['depsgraph'].view_layer
+
+        if view_layer.rpr.crytomatte_aov_material:
+            self._sync_mat_hash()
+
+        if view_layer.rpr.crytomatte_aov_object:
+            self._sync_obj_hash()
+
+    def _sync_obj_hash(self):
+        for key, value in self.objects.items():
+            if not isinstance(value, (pyrpr.Mesh, pyrpr.Instance)):
+                continue
+
+            obj_name = str(key).replace("(", "").replace(")", "").replace(", ", "_").replace("'", "")
```

Comment:
Perhaps this should be @property or just a function?

Reply:
Done.